cmd/vela-docker/repo_test.go

```diff
@@ -10,8 +10,8 @@ func TestDocker_Repo_Validate(t *testing.T) {
 	// setup types
```

Comment:
I believe the name of the directory containing this file needs to be renamed 👍 

Currently it's:

`cmd/vela-docker/repo_test.go`

When I believe it should be:

`cmd/vela-kaniko/repo_test.go`